swift/ql/lib/codeql/swift/elements/expr/LogicalOperation.qll

```diff
@@ -6,31 +6,19 @@ private import codeql.swift.elements.expr.DeclRefExpr
 private import codeql.swift.elements.decl.ConcreteFuncDecl
 
 private predicate unaryHasName(PrefixUnaryExpr e, string name) {
-  e.getFunction()
-      .(DotSyntaxCallExpr)
-      .getFunction()
-      .(DeclRefExpr)
-      .getDecl()
-      .(ConcreteFuncDecl)
-      .getName() = name
+  e.getFunction().(DotSyntaxCallExpr).getStaticTarget().getName() = name
 }
 
 private predicate binaryHasName(BinaryExpr e, string name) {
-  e.getFunction()
-      .(DotSyntaxCallExpr)
-      .getFunction()
-      .(DeclRefExpr)
-      .getDecl()
-      .(ConcreteFuncDecl)
-      .getName() = name
+  e.getFunction().(DotSyntaxCallExpr).getStaticTarget().getName() = name
 }
 
 class LogicalAndExpr extends BinaryExpr {
-  LogicalAndExpr() { binaryHasName(this, "&&") }
+  LogicalAndExpr() { binaryHasName(this, "&&(_:_:)") }
```

Comment:
Thanks for fixing these! I'm surprised it wasn't broken in the CFG tests after I changed the names in https://github.com/github/codeql/pull/9619 🤔.